src/styles.css

```diff
@@ -47,7 +44,6 @@ body {
 
 #twitter-logo {
   width: calc(var(--sizeUnit) * 1.25);
-  display: inline-block;
   position: relative;
   top: calc(var(--sizeUnit) * 0.1);
   padding-right: calc(var(--sizeUnit) * 0.2);
```

Comment:
Allow the color of the SVG Twitter logo to change upon hover: the only way to assign an SVG a new color (when it is called within an `<img>` tag) is via the CSS `filter:` property. This pre-hover value matches the pre-hover grey color used on the "Tweet it!" link (#c9c9c9)